Jenkinsfile

```diff
@@ -12,19 +12,49 @@
       PREVIEW_VERSION = "0.0.0-SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER"
       NODE_OPTIONS = "--max_old_space_size=40000"
       OPTIMIZE_MEMORY = "true"
+        
+
+      API_HOST="http://activiti-cloud-gateway.jx-staging.35.228.195.195.nip.io/modeling-service/"
```

Comment:
Don't we want to make these hosts secret as well? In Travis they are, we were requested to remove the urls from the repo because of security concerns. And this seems to be something which could be configured, without the need of making a commit to the repo.

Reply:
ok